cmd/cx/switch.go

```diff
@@ -16,11 +19,28 @@ func init() {
 
 func runSwitch(c *cli.Context) error {
 	if len(c.Args()) == 0 {
+		sw := *stdcli.DefaultWriter
+
```

Comment:
Another way of getting the current rack is implemented via #290, that way its done through `currentRack()`. Thoughts?